wiki/backup.md

```diff
@@ -1,86 +1,98 @@
 # Backing Up Data
 
-Some updates, software bugs or malware could delete your data, your computer's drive could fail at some point, taking backups is useful when you regularly modify system files or when you format your disks frequently. It's also safer to store data in multiple places in case something happens to your computer's drive.
+There is always the possibility for data loss. It could be updates, software bugs, malware, drive failure, etc. Taking backups of your data is very important. This guide will go over a few ways to back up your data in case something happens to it.
 
 ## What do you need to back up?
 
-It mostly depends on your preference, but you can start with your personal documents, photos, videos. These should be backed up regularly in case you lose them.
+This depends on preference, but it's a good idea to start with personal documents, photos, videos, and any other important files you might have. These backups can then be restored from if you lose the files.
 
 ## The 3-2-1 Backup Rule
 
-The 3-2-1 rule is keeping 3 copies of your data, two on different storage mediums (your computer's drive and a backup drive for example) and one on the cloud.
+The "3-2-1" backup rule is a good guideline to follow when backing up your data. It helps ensure that you always have a copy of your data, even if something happens to a backup.
 
-In other words, if you have a file called "image.jpg" on your computer's main drive, you can create another copy on your external backup drive and/or on the cloud. Now you have 3 copies of image.jpg.
+The 3-2-1 rule is as follows:
+- Keep at least 3 copies of your data
+- Use at least 2 different storage media
+- Keep at least one copy off-site
```

Comment:
maybe use "1" instead of "one" for consistency?

Reply:
Good catch, thanks! Changed